Source/WebCore/PAL/pal/spi/mac/NSSpellCheckerSPI.h

```diff
@@ -54,4 +53,6 @@ extern NSString *NSTextCheckingSuppressInitialCapitalizationKey;
 @end
 #endif
 
+#define _NSTextCheckingTypeSingleCompletion (1ULL << 29)
```

Comment:
We should import the private header underneath `USE(APPLE_INTERNAL_SDK)` and redeclare the enum under `#else`.